### ISSUE TITLE
[1.12.2] White Screen WEBDISPLAY MOD

### ISSUE THREAD

**Issue by rockitosan** [montoyo/webdisplays#56]
My friends can see the screen and navegate on youtube, etc... But i cant´t, only see a white screen please help 

[latest.log](https://github.com/montoyo/webdisplays/files/3667986/latest.log)

**Comment by ItsK1tty** [comment]
Try using VPN

**Comment by montoyo** [comment]
Your OpenGL version is wayyyyyyyy too old for MCEF to work. How old is your graphic card? Can you actually play with it? rofl

**Comment by montoyo** [comment]
Yeah... I'm afraid there's nothing I can do. You really need a more recent graphic card/chipset. Sorry :/

**Comment by ItsK1tty** [comment]
when your gpu doesn't let you to play Minecraft
